ex03.exs

```diff
@@ -60,7 +60,12 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
```

Comment:
## -15

Again, this code doesn't work at all. And the sad thing is that it is a trivial fix.

You have

~~~ elixir
E_C = collection |> Enum.into([])
 E_C 
    |> Enum.chunk_every(div(Enum.count(E_C), process_count))
    |> Enum.map(&(Task.async(fn -> Enum.map(&1, function) end)))
    |> Enum.map(&(Task.await(&1) ))
    |> Enum.concat
~~~

`E_C` is the name of a module, not a variable. It doesn't match the list on the right hand side of the equals sign, so the code fails.

But you didn't need the `E-C` variable anyway. You could have written:

~~~ elixir
  def pmap(collection, process_count, function) do
    collection
    |> Enum.into([])
    |> Enum.chunk_every(div(Enum.count(collection), process_count))
    |> Enum.map(&(Task.async(fn -> Enum.map(&1, function) end)))
    |> Enum.map(&(Task.await(&1) ))
    |> Enum.concat
  end
~~~